### PR TITLE
LPS-134188 Remove x-xss-protection (2)

### PR DESCRIPTION
**Pull request by stian-sigvartsen**

### CODE REVIEW

File: readme/BREAKING_CHANGES.markdown [hunk 1]
```diff
@@ -645,3 +645,34 @@ No manual updates are required.
...
+
+The following methods have been removed:
+
+- com.liferay.portal.kernel.servlet.SanitizedServletResponse#disableXSSAuditor
+- com.liferay.portal.kernel.servlet.SanitizedServletResponse#disableXSSAuditor
+- com.liferay.portal.kernel.servlet.SanitizedServletResponse#disableXSSAuditorOnNextRequest
+- com.liferay.portal.kernel.servlet.SanitizedServletResponse#disableXSSAuditorOnNextRequest
+
+A related constant has been removed also:
+
```
Comment (lipusz) [inline readme/BREAKING_CHANGES.markdown:663]: I think we should also mention the system property that we are removing. :thinking:

Comment (stian-sigvartsen) [reply readme/BREAKING_CHANGES.markdown:663]: Good point. Thanks @lipusz

**Comment by stian-sigvartsen** [comment]
@lipusz please can you check the last commit and ci:forward ?

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
ci:forward

**Comment by stian-sigvartsen** [comment]
@lipusz I'll forward now, but we can always send a followup tweak if you'd like to add/change something to BREAKING_CHANGES.markdown 👍

**Pull request closed without merging**